### DISCUSSION TITLE
Running frontend locally using a deployed instance of superset

### DISCUSSION THREAD

**Discussion by agubler** [apache/superset#23843] [General]
I would like run the frontend locally but proxy the dev server to a deployed instance of superset - I saw instructions in the readme to use `supersetUrl` but this doesn't seem to load the local frontend so the local changes are not respected.

Does anyone have any advice here?

**Comment by EugeneTorap** [comment]
Hi @agubler! Am I correct, you want to develop UI of superset using webpack dev server and proxy all requests to the remote instance?
Does the remote superset deployed on `http` or `https`?
